follow-ups/arrays-objects/index.html

```diff
@@ -0,0 +1,647 @@
+          </p>
+        </section>
+
+        <section>
+          <p>
+            Funkcja <code>renderBook</code> przyjmuje jako jedyny argument obiekt,
+            o następujących wartościach:
+          </p>
+          <ul class="fragment">
+            <li class="fragment"><code>title</code> - tytuł</li>
+            <li class="fragment"><code>author</code> - autor</li>
+            <li class="fragment"><code>year</code> - rok wydania</li>
+          </ul>
+        </section>
+
+        <section>
+          <h3>Zadanie.</h3>
+          <p>
+            Używając naszej funkcji, dodajcie na półkę 5 dowolnych książek.
+          </p>
+        </section>
+
+        <section>
+          <h3>Rozwiązanie</h3>
+          <p>Pewnie wyszło Wam coś takiego:</p>
+          <pre class="fragment"><code class="hljs javascript">let book1 = {
+  author: 'Janina Kowalska',
+  title: 'Mój pamiętnik',
+  year: 2017
+};
+
+renderBook(book1);</code></pre>
+          <p class="fragment">(i tak 5 razy)</p>
+        </section>
+
+        <section>
+          <h3>Prościej</h3>
+          <p>Ale nie trzeba obiektu przypisywać do zmiennej.</p>
+          <p class="fragment">Można go przekazać bezpośrednio:</p>
+          <pre class="fragment"><code class="hljs javascript">renderBook({
+  author: 'Janina Kowalska',
+  title: 'Mój pamiętnik',
+  year: 2017
+});</code></pre>
+        </section>
+
+        <section>
+          <h3>To teraz łączymy</h3>
+          <figure>
+            <img style="height: 400px;" src="./img/merge.png" />
+            <figcaption style="font-size: 0.5rem;"><a href="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4a/MUTCD_W4-1R.svg/2000px-MUTCD_W4-1R.svg.png">
+              Zdjęcie w domenie publicznej.
+            </a></figcaption>
+          </figure>
+          <p>Obiekty i tablice razem.</p>
+        </section>
+
+        <section>
+          <h3>Przypomnijmy sobie</h3>
+          <ul>
+            <li class="fragment">W zmiennej można przechowywać tablicę albo obiekt</li>
+            <li class="fragment">Tablica to zbiór ponumerowanych zmiennych</li>
+            <li class="fragment">Obiekt to zbiór nazwanych zmiennych</li>
+          </ul>
+          <p class="fragment">więc...</p>
+        </section>
+
+        <section>
+          <h3>Możemy je zagnieżdżać w sobie</h3>
+        </section>
+
+        <section>
+          <h3>Tablice w tablicy</h3>
+          <pre class="fragment"><code class="hljs javascript"> let ultraArray = [
+  [1, 2, 3],
+  ['red', 'blue', 'green'],
+  ['small', 'medium', 'big']
+];</code></pre>
+          <p class="fragment">Aby dostać się do wartości:</p>
+          <pre class="fragment"><code class="hljs javascript">ultraArray[1][0] // 'red'</code></pre>
+        </section>
+
+        <section>
+          <h3>Tablice w tablicy</h3>
+          <p class="note side-note">
+            Tablice w tablicy są raczej rzadko spotykane, ze względu na
+            problemy z czytelnością.
```

Comment:
być może nieco za bardzo po macoszemu tutaj traktujemy temat wielowymiarowości tablic? może możnaby jednak użyć jakiegoś przykładu tablicy dwuwymiarowej jako planszy do gry, czy coś, i pokazać na tej podstawie że taka tablica dwuwymiarowa to w rzeczywistości tablica tablic...

Reply:
Właśnie nie wiem, czy jest sens? Chciałem tylko zaznaczyć, że jest taki twór, ale nie wdawać się w szczegóły. Obawiam się, że jak zaczniemy tłumaczyć tablice 2d, to uczestniczki będą się gubić.

Reply:
OK, ale jeśli chcesz powiedzieć, że zagadnienie tablic wielowymiarowych jest poza zakresem spotkania,
to komentarz "Tablice w tablicy są raczej rzadko spotykane, ze względu na problemy z czytelnością." jest trochę słaby.